proteus/MeshTools.py

```diff
@@ -6001,3 +6001,75 @@ def getMeshIntersections(mesh, toPolyhedron, endpoints):
                 continue
             intersections.update(((tuple(elementIntersections[0]), tuple(elementIntersections[1])),),)
     return intersections
+
+
+from proteus import default_n as dn
+from proteus import default_p as dp
+
+class MeshOptions:
+    """
+    Mesh options for the domain
+    """
+    def __init__(self, domain=None):
+        self.Domain = domain
+        self.he = 1.
+        self.use_gmsh = False
```

Comment:
Do you mean having something such as `p.domain.MeshOptions.use_gmsh` instead of `p.domain.use_gmsh` or have it completely separated as to something such as `p.MeshOptions.use_gmsh`? (the latter will break things in current tests as it requires the p files to have a `MeshOptions` class instance).
https://github.com/erdc-cm/proteus/blob/add_gmsh_support/proteus/NumericalSolution.py#L242